ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/LedgerApiServer.scala

```diff
@@ -52,136 +65,99 @@ object LedgerApiServer {
       poolName = s"ledger-api-server-rs-grpc-bridge-${UUID.randomUUID}",
       actorCount = Runtime.getRuntime.availableProcessors() * 8
     )(mat.system)
+    closeables.push(serverEsf)
+
+    val workerEventLoopGroup = createEventLoopGroup(
+      mat.system.name + "-nio-worker",
+      parallelism = Runtime.getRuntime.availableProcessors)
+    val bossEventLoopGroup = createEventLoopGroup(mat.system.name + "-nio-boss", parallelism = 1)
+    closeables.push(() => {
```

Comment:
`closeables` is a stack of `AutoCloseable`s, how come you're passing a function here (and in other places)?

Reply:
The Scala compiler can turn a function into a functional interface.

Reply:
Uh, of course, thanks. :slightly_smiling_face: